packages/ffe-datepicker-react/src/calendar/Header.js

```diff
@@ -1,50 +1,139 @@
 import React from 'react';
 import { func, number, string } from 'prop-types';
-import ChevronIkon from '@sb1/ffe-icons-react/lib/chevron-ikon';
+import { SecondaryButton } from '@sb1/ffe-buttons-react';
+import Dropdown from '@sb1/ffe-dropdown-react';
+import MonthPicker from './MonthPicker';
+import i18n from '../i18n/i18n';
 
 export default function Header(props) {
     const {
         datepickerId,
         month,
+        year,
+        language,
         nextMonthHandler,
-        nextMonthLabel,
         previousMonthHandler,
-        previousMonthLabel,
-        year,
+        todayHandler,
+        selectYearHandler,
     } = props;
 
+    function renderYearOptions() {
+        const distribution = 5;
+        const minYear = year - distribution;
+        const maxYear = year + distribution;
+        const options = [];
+        let currentYear = minYear;
+        let option;
+
+        while (currentYear <= maxYear) {
+            option = (
```

Comment:
Any reason to store this in a variable like this?
Seems like you could just
```
options.push(
  <option />
);
```

Reply:
Det er ingen spesiell grunn. Jeg satte opprinnelig selected=true/false på option elementene, men dette ble endret. Nå settes value på select elementet istedet.